### PR TITLE
[Search page]: "Less filters" button should be on one line

### PR DESCRIPTION
**Pull request by cmoinier**
### Description

This PR changes the styling of the "less filters" button on the search page to keep its text on one line.

### Screenshot

image

### CODE REVIEW

File: apps/datahub/src/app/search/search-filters/search-filters.component.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
   </div>
   <div
     class="flex flex-row items-start gap-4 -ml-[2px] mt-2 justify-betwwen w-56"
+    [ngClass]="{ 'w-60': displayCount === searchConfig.length }"
```
Comment (tkohr) [inline apps/datahub/src/app/search/search-filters/search-filters.component.html:25]: Couldn't we just raise the `w-56` above to `w-60`? 

Btw, I'm also noticing that this area of the search filters is not responsive at all, but this seems to be out of scope for this PR I guess, as responsiveness has not yet been a real requirement.

Comment (cmoinier) [reply apps/datahub/src/app/search/search-filters/search-filters.component.html:25]: Yes, we totally can, I made the change!

**Review by tkohr — APPROVED** [review]
Thanks for the fix @cmoinier !
